src/actions/mentors.js

```diff
@@ -0,0 +1,25 @@
+export const FETCH_MENTOR_START = "FETCH_MENTOR_START";
+export const FETCH_MENTOR_SUCCESS = "FETCH_MENTOR_SUCCESS";
+export const FETCH_MENTOR_FAILURE = "FETCH_MENTOR_FAILURE";
+
+// export const fetchMentors = () => {
+//   return (dispatch, getState, { getFirestore, getFirebase }) => {
+//     dispatch({ type: FETCH_MENTOR_START });
+
+//     const firestore = getFirestore(); //can now access user collection in firestore
+
```

Comment:
I commented out the actions since we're not getting data from firebase yet with the mentors listed.